src/config/dynamic-theme-fixes.config

```diff
@@ -14267,6 +14267,18 @@ body {
 
 ================================
 
+lists.debian.org
```

Comment:
Good catch but as I checked the debian.org fixes we repeat fixes from there here.
The same problem I found on wiki.debian.com

So instead of making fix for subpages with prefixes I propose you add alias `*.debian.org` under `debian.org` line in sitefixes.

Reply:
I didn't realise that was possible! Thanks for the tip, I'll make the changes now.